parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -25,6 +25,47 @@ installDeps() {
     fi
 }
 
+installKataDeps() {
+    if [[ $OS_VERSION == "2.0" ]]; then
+      for dnf_package in cargo opa parted qemu-img moby-runc python3-devel python3-pip kernel-mshv cloud-hypervisor kata-containers moby-containerd-cc mshv-bootloader mshv-linuxloader mshv; do
```

Comment:
1. how about an array or something? anything where we define this list properly
2. for these packages, why do we need to list those explicitly resp., install those: cargo opa parted qemu-img moby-runc python3-devel python3-pip

Reply:
1. The general AgentBaker pattern I observed doesn't define list for package dependencies. I think creating one just above the for loop wouldn't be much different. But I agree this part needs a bit of improvement in terms of clarity. Just added a comment to help with that.
2. Good catch! That's a side-effect of installing storage account packages through the RPM tool. Removed non-kata packages here.